chromium/background.js

```diff
@@ -708,8 +612,7 @@ async function import_settings(settings) {
 
 Object.assign(exports, {
   all_rules,
-  initializeStoredGlobals,
-  urlBlacklist,
+  urlBlacklist
```

Comment:
I usually leave a trailing `,` on the final entry because it makes diffs less noisy when you append an entry later. NBD tho.

Reply:
I like this way better, syntactically.